bql/planner/planner_test.go

```diff
@@ -778,7 +808,7 @@ func TestPlannerQuery(t *testing.T) {
 		// Actual test:
 		tbl, err := plnr.Execute(ctx)
 		if err != nil {
-			t.Fatalf("planner.Execute(%s)\n= _, %v; want nil error", entry.q, err)
+			t.Fatalf("planner.Execute(%s)\n= _, %v; want _, nil error", entry.q, err)
```

Comment:
`planner.Execute(%s)\n= _, %v; want _, nil` is enough here - no need to add the `error`, as you already specify it's the second input you're comparing. It doesn't really matter what the type of the second return is, just that you got something different from what you expected.

Same for the other messages below. :)

Reply:
Done. Thank you Tati. =]